orchagent/fgnhgorch.cpp

```diff
@@ -686,12 +775,21 @@ bool FgNhgOrch::setActiveBankHashBucketChanges(FGNextHopGroupEntry *syncd_fg_rou
         uint32_t num_nhs_with_one_more = (num_buckets_in_bank % total_nhs);
         uint32_t num_nhs_with_eq_to_exp = total_nhs - num_nhs_with_one_more;
         uint32_t add_nh_exp_bucket_size = exp_bucket_size;
+        uint32_t exp_size_before_bkt_rem = exp_bucket_size;
```

Comment:
remove variable and use exp_bucket_size itself

Reply:
Done.